src/osemgrep/reporting/Output.ml

```diff
@@ -150,8 +150,11 @@ let dispatch_output_format (output_format : Output_format.t) (conf : conf)
         runtime_params.is_logged_in || is_pro
         || not runtime_params.is_using_registry
       in
+      (* TODO: use the config flag to determine whether to show dataflow traces. *)
```

Comment:
use the config indeed.
Why not implementing it here?

Reply:
I was hoping to implement it here (at this location), but not here (in this PR).

If it's an easy 1-5 LOC and makes an existing osemfail test pass, I'll include in this PR.

Reply:
It was a small change. Added it.